data/l10n/en/main.ftl

```diff
@@ -47,5 +47,12 @@ gateway-on-look-locked = It is locked.
 void-name = the void
 void-description = a place filled with nothingness
 shell-invalid-command = This is not a valid command.
-quit-game-message = This is the end.
+quit-game-message-light = some light on the horizon, the first dancing rays of the sun.
```

Comment:
These are not whole sentences. Is that fine?

Reply:
they are meant to complete "You see:" as in:

```
> quit
You are now in the void. You see:
This is the end.
```

```
void-name = the void
void-description = a place filled with nothingness
quit-game-message = This is the end.
 ```
 
 We could also change the endings to complete sentences and strip the "You see:" part.